### PR TITLE
Updating Master

### PR DESCRIPTION
**Pull request by gbanares**

### CODE REVIEW

**Comment by gbanares** [comment]
cleaning up files from last dev to master push.

File: README.md [hunk 1]
```diff
@@ -10,8 +10,7 @@ Created for project 3. Details to follow.
 
 1. Clone Repo
 1. Run npm i
-1. Create a config folder
-    1. Copy config.template and save as config.json
+1. Copy contents of config.template and save as .env
```
Comment (jmckenney) [inline README.md]: Nice working keeping this up to date as you make changes.

File: client/src/components/Shop/productList.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React, { Component } from "react";
+// TODO: Create products db to hold prod data
+// FIXME: import API from "../../utils/API";
```
Comment (jmckenney) [inline client/src/components/Shop/productList.js]: Worthwhile discussion as to the merits of inline todos vs tickets/issues in the project backlog.

**Comment by gbanares** [comment]
Closing and will update with new PR

**Pull request closed without merging**